services/graph.go

```diff
@@ -0,0 +1,17 @@
+package services
+
+import (
+	"encoding/json"
+	"errors"
+
+	"github.com/Los-Crackitos/Excelante/models"
+)
+
+func (file *File) createGraph(sheetName string, startingCoordinates string, graph *models.Graph) error {
+	graphProps, err := json.Marshal(graph)
+	if err != nil {
+		return errors.New("An error occurred when decode graph JSON data")
```

Comment:
FYI, `json.Marshal` transform a value into JSON thus the error text is wrong but nevermind this shouldn't be here. You are transforming a value that has been already [decoded before ](https://github.com/Los-Crackitos/Excelante/blob/e4956109d6fb3612aaf93e4dee3e55ec023fc7fb/handlers/writer.go#L28) by the JSON decoder using the correct model then, I think, we will never pass through this condition and show this error, we can ignore the `err` value that will be `nil` every time and remove this check.